emloop/utils/config.py

```diff
@@ -41,9 +46,13 @@ def load_config(config_file: str, additional_args: typing.Iterable[str]=()) -> d
         key = key_split[-1]
 
         conf = config
-        for key_part in key_prefix:
-            conf = conf[key_part]
-        conf[key] = value
+        override_value(conf, key, key_prefix, value)
+
+        if 'eval' in config:
+            eval_conf = config['eval']
+            for stream in eval_conf:
```

Comment:
I do not get this!

We **do not want to always override all the `eval.*` options**. Moreover, it should be done **before** the cli options.

I.e.:
- load the config
- if specified, override the options from selected `eval.something` section
- override the options from cli args